src/main/scala/cromwell/engine/callactor/FinalCallActor.scala

```diff
@@ -0,0 +1,14 @@
+package cromwell.engine.callactor
+
+import akka.actor.ActorRef
+import cromwell.engine.WorkflowDescriptor
+import cromwell.engine.callexecution.CallExecutionActor
```

Comment:
I sort-of agree with you both... I'll leave this as-is for now since it'll all be removed and placed somewhere else imminently.